### PR TITLE
add Sentry

### PR DESCRIPTION
**Pull request by jipiboily**
@rainforestapp/customer-product can I haz review plz?

### CODE REVIEW

File: config/initializers/sentry-raven.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'raven'
+
+Raven.configure do |config|
+  config.dsn = ENV.fetch('INTEGRATION_SENTRY_DNS')
```
Comment (epaulet) [inline config/initializers/sentry-raven.rb]: Why is one DSN and the other DNS?

Comment (jipiboily) [inline config/initializers/sentry-raven.rb]: huh, that's a good question, I'm pretty sure I copy pasted the `config.dsn` part and it looks wrong. Double checking now.

Comment (jipiboily) [inline config/initializers/sentry-raven.rb]: That's my bad, I misread it, it's DSN...changing. Good catch!

**Comment by jipiboily** [comment]
@epaulet fixed, and I've unset the bad one and set the right one in staging and prod.

**Comment by epaulet** [comment]
merge conflicts, but otherwise lgtm

**Pull request merged**